models/modelling/commissioning/encounters/int_mhsds_spell_encounters.sql

```
@@ -7,33 +7,181 @@ Clinical Purpose:
+        , case when fiscal_year_start = min(fiscal_year_start) over ()
+            then '1900-01-01'::date
+            else date_from_parts(fiscal_year_start, 4, 1)
+        end as fy_range_start
+        , case when fiscal_year_start = max(fiscal_year_start) over ()
+            then '2099-12-31'::date
+            else date_from_parts(fiscal_year_start + 1, 3, 31)
+        end as fy_range_end
+    from {{ ref('uk_cost_indices') }}
+)
+
+, price_base_deflator as (
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)